cdap-ui/app/tracker/controllers/tracker-enable-ctrl.js

```diff
@@ -169,6 +169,12 @@ class TrackerEnableController{
       return;
     }
 
+    if (this.$state.params.sourceUrl) {
+      window.location.href = decodeURIComponent(this.$state.params.sourceUrl);
+    } else {
+      this.$state.go('tracker.home');
+    }
+
     this.$state.go('tracker.home');
```

Comment:
Unnecessary now with the conditional added, or keep this and remove the `else` part?